Sources/Amplitude/Plugins/Vendors/AppUtil.swift

```diff
@@ -28,14 +28,24 @@ import Foundation
         }
 
         override var os_name: String {
-            return device.systemName
+            return device.systemName.lowercased()
```

Comment:
We are using lower cases for os_name before.
https://github.com/amplitude/Amplitude-iOS/blob/main/Sources/Amplitude/AMPConstants.m#L46